apps/ledger-live-mobile/src/screens/Manager/Benchmarking.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import styled from "styled-components/native";
+import { useAppInstallProgress } from "@ledgerhq/live-common/apps/react";
+import Config from "react-native-config";
+
+const Wrapper = styled(Flex).attrs({
+  bg: "palette.background.default",
+  p: 4,
+  mt: 4,
+  borderWidth: 1,
+  borderRadius: 4,
+  borderColor: "neutral.c40",
+})``;
+
+const Status = styled(Flex).attrs(p => ({
+  bg: p.active ? "success.c100" : "warning.c100",
+  borderRadius: 20,
+  borderWidth: 0,
+  height: 10,
+  width: 10,
+}))``;
+
+export default function Benchmarking(props: any) {
```

Comment:
`props` should be typed to `{ state: State }`, no ?

Reply:
Everything should be typed properly, but State from device actions is already _any_ no?

Reply:
Actually, it looks like from `AppsScreen.tsx` that `State` is typed:

```TS
import { distribute, Action, State } from "@ledgerhq/live-common/apps/index";

// From live-common/apps/types.ts
export type State = {
  deviceInfo: DeviceInfo;
  deviceModel: DeviceModel;
  firmware: FinalFirmware | null | undefined;
  appByName: Record<string, App>;
  apps: App[];
  installedAvailable: boolean;
  installed: InstalledItem[];
  recentlyInstalledApps: string[];
  installQueue: string[];
  uninstallQueue: string[];
  // queue saved at the time of a "updateAll" action
  updateAllQueue: string[];
  currentAppOp: AppOp | null | undefined;
  currentProgressSubject: Subject<number> | null | undefined;
  currentError:
    | {
        error: Error;
        appOp: AppOp;
      }
    | null
    | undefined;
};
```